models/product/src/product-variant/product-variant-draft/presets/sample-data-goodstore/aria-rug-01.ts

```diff
@@ -0,0 +1,71 @@
+import { Money, PriceDraft } from '@commercetools-test-data/commons';
+import { AttributeDraft } from '../../../../attribute';
+import * as Image from '../../../../image';
+import type { TProductVariantDraftBuilder } from '../../../types';
+import * as ProductVariantDraft from '../../index';
+
+const ariaRug01 = (): TProductVariantDraftBuilder =>
+  ProductVariantDraft.presets
+    .empty()
+    .sku('AAR-34')
+    .prices([
+      PriceDraft.presets
+        .empty()
+        .value(Money.random().currencyCode('EUR').centAmount(12499))
+        .country('DE'),
+      PriceDraft.presets
+        .empty()
+        .value(Money.random().currencyCode('GBP').centAmount(12499))
+        .country('GB'),
+      PriceDraft.presets
+        .empty()
+        .value(Money.random().currencyCode('USD').centAmount(12499))
+        .country('US'),
+    ])
+    .images([
+      Image.ImageDraft.presets
```

Comment:
This is different from how we normally import `Draft`

Reply:
Addressed in a060061fbc7d8a0f2a55598d4eab2e6162b8af71